src/Kernel32/Kernel32.cs

```diff
@@ -157,6 +157,14 @@ public static partial class Kernel32
         [DllImport(api_ms_win_core_processthreads_l1_1_1)]
         public static extern int GetCurrentProcessId();
 
+        /// <summary>
+        /// Retrieves the process identifier of the specified process.
+        /// </summary>
+        /// <param name="Process">A handle to the process. The handle must have the PROCESS_QUERY_INFORMATION or PROCESS_QUERY_LIMITED_INFORMATION access right. For more information, see <see cref="https://msdn.microsoft.com/de-de/library/windows/desktop/ms684880(v=vs.85).aspx"/></param>
```

Comment:
You evidently can't use `<see cref` for a URL as it generates a compiler warning. I recommend you just drop the tag and leave the URL.

Reply:
I removed that in my local project and just put the name of the site there instead of the full URL.
Can I somehow edit this pull request instead of creating new one? I am not familiar with Git/PullRequests yet.